ajax/libs/sticky-js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sticky-js",
+  "filename": "dist/sticky.min.js",
```

Comment:
Let's simply use `sticky.min.js`. You can see `basePath` as the root, therefore we only need `sticky.min.js`.